atom/atom.py

```diff
@@ -117,7 +117,11 @@ def __init__(
 
     def __call__(
         self,
-        func: Union[Callable[[ChangeDict], None], Callable[[T, ChangeDict], None]],
+        func: Union[
```

Comment:
Looking at the implementation I don't see how it works. Could you point me to an example in the enaml codebase ?

Reply:
If you uncomment the "ObserverHandler" then run `mypy examples` it fails when calling the super method.